resources/views/components/js/merge.blade.php

```diff
@@ -1,26 +1,11 @@
 <script>
-    var mergin = false;
-    function onMergePressed(){
-        if( ! mergin) {
-            return startMergin();
-        }
-        merge();
-    }
-
-    function startMergin(){
-        mergin = true;
-        $("#mergeButton").removeClass("secondary");
-        $("#mergeButton").html("{{__('ticket.mergeDesc')}}");
-        $(".selector").show();
-    }
-
     function getSelectedTicketsIds(){
         return $("input[name^=selected]:checked").map(function() {
```

Comment:
Looks like this getSelectedTicketsIds is duplicated now?

Reply:
correct, fixed it by merging the merge and status functions together.